pkg/volume/cinder/provisioner/iscsi_test.go

```diff
@@ -74,7 +73,7 @@ var _ = Describe("Iscsi Mapper", func() {
 
 		JustBeforeEach(func() {
 			secretName = getChapSecretName(conn, options)
-			source, err = mapper.BuildPVSource(conn, options)
```

Comment:
The idea of the lint is that ignoring the error is a bug - not that you should just ignore the error in a different way :-P

Gingko makes this easy with an assertion like:
`Expect(err).NotTo(HaveOccurred())`

Reply:
Gotcha. Had a bad feeling about just dropping all those errors, just wasn't sure how to go about it. Thanks.